### PR TITLE
CB-5332 fix multiple termination trigger

### PR DESCRIPTION
**Pull request by lacikaaa**

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/UpdateHostsValidator.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
+package com.sequenceiq.cloudbreak.service.cluster.flow;
```
Comment (keyki) [inline core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/UpdateHostsValidator.java]: I'm not sure what would be a good name, but I don't like something being called *FlowService and the methods are not really related to flow operations.

File: core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/ClusterFlowService.java [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+import com.sequenceiq.cloudbreak.structuredevent.event.CloudbreakEventService;
+import com.sequenceiq.cloudbreak.template.validation.BlueprintValidator;
+import com.sequenceiq.cloudbreak.util.UsageLoggingUtil;
+import com.sequenceiq.cloudbreak.workspace.model.User;
+import com.sequenceiq.cloudbreak.workspace.model.Workspace;
+import com.sequenceiq.common.api.type.InstanceGroupType;
+import com.sequenceiq.common.api.type.ResourceType;
+
+@Service
+public class ClusterFlowService {
```
Comment (keyki) [inline core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/ClusterFlowService.java]: I'm not sure what would be a good name, but I don't like something being called *FlowService and the methods are not really related to flow operations.

Comment (lacikaaa) [reply core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/ClusterFlowService.java]: renamed

**Comment by lacikaaa** [comment]
retest this please

**Pull request closed without merging**